script/deploy/healthCheck.ts

```diff
@@ -79,7 +80,7 @@ const main = defineCommand({
         !coreFacets.includes(k) &&
         !corePeriphery.includes(k) &&
         k !== 'LiFiDiamond' &&
-        k.endsWith('Facet')
+        k.includes('Facet')
```

Comment:
quick question: why does it change from `endsWith` to `includes`

Reply:
cause we have Facets that do not end with "Facet" e.g. AcrossFacetPackedV3

Reply:
true! thanks!